packages/dataviz/.storybook/preview.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import IconsProvider from "@talend/react-components/lib/IconsProvider";
```

Comment:
Do you use icons?

Reply:
right now? no.. but I forgot to replace blank spaces with the svg.. on it